docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -2549,8 +2573,43 @@ Default stratifications to all observers should include scenario and input draw.
       * Confirm that neonatal mortality calibration was not worsened relative to prior model run (as this change may affect the LBWSG PAF values)
       * Confirm expected effects of IFA and MMS on pregnancy outcomes (note this will be confounded by ANC attendance in the simulation outputs, so RRs should be calculated stratified by ANC attendance exposure)
       * Confirm expected effects of misoprostol and azithromycin interventions on maternal disorders using scenarios #12 and #13
-    - 
-    - 
+    - * Zero neonatal deaths
+      * Did not yet check other criteria
+    - V&V notebooks included in `this PR <https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/pull/168>`__
+  * - 24.1
+    - MMS stillbirth effects and GA floors with neonatal deaths bugfix
+    - Same as 24.0
+    - * Neonatal mortality looks good again
+      * GA floors do not appear to vary by pregnancy outcome
+      * Did not yet check other criteria
+    - V&V notebooks included in `this PR <https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/pull/168>`__
+  * - 24.2
+    - MMS stillbirth effects and GA floors with neonatal deaths bugfix and re-generated RR caps
+    - Same as 24.0
+    - * Neonatal mortality unchanged from 24.1 -- this run was unnecessary, we hadn't realized that the RR cap calculation doesn't use interpolated relative risks
+      * MMS on stillbirth effects appear correct
+      * Misoprostol and azithromycin effects appear correct
+      * GA floors still not fixed (as expected)
+    - V&V notebooks included in `this PR <https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/pull/168>`__
+  * - 25.0
+    - Update SBR to >=24 weeks
+    - * Stillbirth ratio in simulation still matches artifact
+      * Stillbirth ratio slightly decreased relative to previous model
```

Comment:
```suggestion
      * Stillbirth ratio slightly decreased relative to previous model, as expected
```

Reply:
Note: this is the V&V criteria, not results. Maybe:

```suggestion
      * Stillbirth ratio should be slightly decreased relative to previous model
```